client/apps/edit/components/content/sections/text/index.coffee

```diff
@@ -333,6 +348,10 @@ module.exports = React.createClass
           keyBindingFn: keyBindingFnFull
           handlePastedText: @onPaste
           blockRenderMap: blockRenderMap()
+          handleReturn: @handleReturn
+          onTab: @handleTab
+          onUpArrow: @handleChangeSection
+          onDownArrow: @handleChangeSection
```

Comment:
The fact that this file is now approaching 350 lines is a good sign that things should be extracted out!